packages/journalforing/src/kodeverk/dokumentTittel.ts

```diff
@@ -33,6 +33,16 @@ export enum DokumentTittel {
   RETTSKJENNELSE_TRYGDERETTEN = 'Rettskjennelse fra Trygderetten',
 }
 
+const tittlerSomTillaterOpprettelseAvSak: DokumentTittel[] = [DokumentTittel.SØKNAD_FORELDREPENGER_ENDRING_KVOTE,
```

Comment:
Skal ikke kunne opprette fra SØKNAD_FORELDREPENGER_ENDRING_KVOTE
Men kan tillate "Inntektsmelding". 

Er det ikke mer lekkert å la backend send med et flagg kanOppretteSak på journalist? Da samler vi logikken for dokument vs opprettSak ett sted. 
Subsidiært sjekke !!journalpost.ytelsetype (eller hvor mange ! man trenger)

Reply:
Gikk over til å sjekke det boolske fra backend, så da er vel ikke dette relevant lenger